fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallViewModel.java

```diff
@@ -142,11 +142,17 @@ public Completable openRollCall(String id) {
       ErrorUtils.logAndShow(getApplication(), TAG, e, R.string.unknown_lao_exception);
       return Completable.error(new UnknownLaoException());
     }
+
+    if (!rollCallRepo.canOpenRollCall(laoId)) {
+      Log.d(TAG, "failed to open roll call with id " + id + ", laoID: " + laoView.getId());
```

Comment:
The log message could be more precise (failed because another roll call was already open)